Dockerfile-authenticator

```diff
@@ -11,7 +11,7 @@ ENV LC_ALL en_US.utf-8
 ENV LANG en_US.utf-8
 RUN pipenv install
 ADD users/ /opt/arxiv/users/
-RUN pipenv install /opt/arxiv/users/
+RUN pipenv install /opt/arxiv/users/ --skip-lock
```

Comment:
What is the intent of skipping the lock file?

Reply:
Oh probably not needed anymore, was running into some lock shenanigans